src/Components/SmartComponents/HybridInventory/ImmutableDevicesTab/helpers.js

```diff
@@ -0,0 +1,80 @@
+import { getAffectedSystemsByCVE, useGetImageData } from '../../../../Helpers/APIHelper';
```

Comment:
There is a lot of code duplication here with regular Systems table, do you plan to merge the code somehow in the future?